winlogbeat/eventlog/factory.go

```diff
@@ -10,7 +10,8 @@ import (
 	"github.com/elastic/beats/libbeat/common"
 )
 
-var commonConfigKeys = []string{"api", "name", "fields", "fields_under_root", "tags"}
+var commonConfigKeys = []string{"api", "name", "fields", "fields_under_root",
```

Comment:
Kind of nice to have this list for error checking but requires extendig it every time we add a global option.